src/vectorclock.erl

```diff
@@ -130,6 +104,15 @@ set_clock_of_dc(DcId, Time, VectorClock) ->
 from_list(List) ->
     dict:from_list(List).
 
+max([]) -> dict:new();
```

Comment:
Hi, just to know. This has not been merged, right?
I want to include it in the eventually consistent branch.


> On Dec 8, 2015, at 16:06, Annette Bieniusa <notifications@github.com> wrote:
> 
> In src/vectorclock.erl <https://github.com/SyncFree/antidote/pull/174#discussion_r46963551>:
> 
> > @@ -130,6 +104,15 @@ set_clock_of_dc(DcId, Time, VectorClock) ->
> >  from_list(List) ->
> >      dict:from_list(List).
> >  
> > +max([]) -> dict:new();
> max() and min() need unit tests.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/174/files#r46963551>.
>